### DISCUSSION TITLE
v0.13.0

### DISCUSSION THREAD

**Discussion by steebchen** [prisma/prisma-client-go#715] [General]
# v0.13.0

Bugfixes, bump Prisma CLI, docs

🌟 **Help us spread the word about Prisma by starring the repo ☝️ or [tweeting](https://twitter.com/intent/tweet?text=Check%20out%20the%20latest%20Prisma%20Go%20Client%20release%20v0.13.0%20%F0%9F%9A%80%0D%0A%0D%0Ahttps://github.com/prisma/prisma-client-go/releases/tag/v0.13.0) about the release.**

## Changes

* refactor: use `io` and `os` instead of `io/ioutil` (#653) @s-takehana
* docs(readme): add badges (#654) @s-takehana
* chore(idea): use new log env var in test config (#656) @steebchen
* fix(builder): re-use fields, remove AND wrapping (#657) @steebchen
* test(types/json): add json filter tests (#659) @steebchen
* feat(proxy): add data proxy engine implementation & test (#598) @steebchen
* chore(project): enable dependabot version updates (#641) @s-takehana
* feat(prisma): upgrade prisma to 3.4.0 (#671) @steebchen
* chore(deps): bump golang from 1.16 to 1.17.2 in /test/integration (#672) @dependabot
* chore(deps): bump github.com/stretchr/testify from 1.4.0 to 1.7.0 (#675) @dependabot
* ci(e2e): enable verbose logging (#677) @steebchen
* fix(proxy): introduce retryableRequest, minor fixes (#678) @steebchen
* feat(client): add order by relevance (#660) @steebchen
* chore(deps): bump actions/checkout from 1 to 2.4.0 (#673) @dependabot
* chore(deps): bump amannn/action-semantic-pull-request from 1.1.1 to 3.4.6 (#674) @dependabot
* chore(deps): bump github.com/shopspring/decimal from 1.2.0 to 1.3.1 (#676) @dependabot
* chore(deps): bump github.com/joho/godotenv from 1.3.0 to 1.4.0 (#679) @dependabot
* fix(builder): remove log (#683) @steebchen
* docs(reference/json): add json filter docs (#682) @steebchen
* fix(builder): wrap order by inputs in list (#685) @steebchen
* chore(deps): bump golang from 1.17.2 to 1.17.3 in /test/integration (#680) @dependabot
* refactor(tests): replace word 'fail' in tests (#686) @steebchen
* fix(generator): don't generate gitignores w/o binaries (#687) @steebchen
* chore(idea): set new log variable in e2e test config (#689) @steebchen
* docs(proxy): add data proxy docs (#690) @steebchen
* docs(features): add case sensitivity docs (#691) @steebchen
* docs(features): add index readme (#692) @steebchen
* test(mongodb): add mongodb integration tests (#693) @steebchen
* feat(generator): export query builder interfaces (#694) @steebchen
* test(generator): skip generation on db push (#695) @steebchen
* chore(deps): bump golang from 1.17.3 to 1.17.4 in /test/integration (#700) @dependabot
* chore(deps): bump amannn/action-semantic-pull-request from 3.4.6 to 3.5.0 (#703) @dependabot
* docs(readme): add note about maintenance status (#708) @matthewmueller
* chore(deps): bump golang from 1.17.4 to 1.17.5 in /test/integration (#701) @dependabot
* chore(prisma): upgrade prisma to 3.7.0 (#709) @steebchen
* docs(readme): adapt deprecation notice (#711) @nikolasburk
* chore(deps): bump golang from 1.17.5 to 1.17.6 in /test/integration (#713) @dependabot
* docs(contributing): add CONTRIBUTING.md and more READMEs (#710) @steebchen

## Contributors

@dependabot, @dependabot[bot], @matthewmueller, @nikolasburk, @s-takehana and @steebchen

<hr /><em>This discussion was created from the release <a href='https://github.com/prisma/prisma-client-go/releases/tag/v0.13.0'>v0.13.0</a>.</em>

**Comment by Jovenspier** [comment]
Hi, is this supposed to mean we can continue to use it, can we have hope for the future please?

**Reply by steebchen** [reply]
Hey, the company's decision was to shut it down or pause it until further notice. I just personally cut this release and will try to bump the Prisma CLI version from time to time, but this is on a voluntary basis and there's no guarantee stuff will be fixed.

**Reply by Jovenspier** [reply]
Ok thank you, what do you advise us to do? personally, I keep watching this Repo every day to see the updates, can we hope to have a stable version to use in production?

**Reply by steebchen** [reply]
The good thing is that as long as the Prisma binaries are kept up to date (which I will to voluntarily for the foreseeable future), the queries you write usually work flawlessly, and bugfixes and improvements in the Prisma CLI and the engines will still propagate to the Go client. However, note that new features in the TypeScript client might not be available in the Go client. In my personal opinion it's stable enough for small to medium projects, but if you expect extremely high load or have a million dollar budget project behind it would maybe be better to use something maintained. Hope that helps :)

**Reply by Jovenspier** [reply]
Thanks for your help, you are doing a great job for the community, I hope the Prisma team will change their minds.
